src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,35 @@
+@import "../utils/variables";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  grid-template-rows: repeat(7, $card-heigth);
```

Comment:
<img width="1440" alt="Screenshot 2022-11-18 at 10 31 23" src="https://user-images.githubusercontent.com/94289038/202656876-8ef432e1-30c3-4fd0-86dd-82fa122db1dc.png">
* why do you need predefine such gig amount of rows
I have scroll now - fix it